.circleci/install_circle_cli.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Install the CircleCI CLI tool.
+# https://github.com/CircleCI-Public/circleci-cli
+#
+# Dependencies: curl, cut
+#
+# The version to install and the binary location can be passed in via VERSION and DESTDIR respectively.
+#
+
+set -o errexit
+
+echo "Starting installation."
+
+# GitHub's URL for the latest release, will redirect.
+LATEST_URL="https://github.com/CircleCI-Public/circleci-cli/releases/latest/"
+DESTDIR="${DESTDIR:-$HOME/.local/bin/}"
```

Comment:
On MacOS that very easily might not be in `PATH`. Would it be too complicated to check for `brew` and install `circleci` through it?

Reply:
Hmm, no not too bad.  This is an official install script from the vendor, however.

Reply:
The problem is that it doesn't integrate well with the rest of the system. Perhaps we can at least check if it's already present? If I were using Linux, I'd prefer to use a repo version as well.

Reply:
Yeah, perhaps it's a bit too invasive. I'm leaning towards removing it.

Reply:
:heavy_check_mark: - removed.